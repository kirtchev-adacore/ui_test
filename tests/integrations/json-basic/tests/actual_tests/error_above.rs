use json_basic::add;

fn main() {
    //~v ERROR: mismatched types
    add("42", 3);
}
