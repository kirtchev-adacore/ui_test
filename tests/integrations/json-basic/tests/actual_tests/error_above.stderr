error[E0308]: mismatched types
 --> tests/actual_tests/error_above.rs:5:9
  |
5 |     add("42", 3);
  |     --- ^^^^ expected `usize`, found `&str`
  |     |
  |     arguments to this function are incorrect
  |
note: function defined here
 --> $DIR/tests/integrations/json-basic/src/lib.rs:1:8
  |
1 | pub fn add(left: usize, right: usize) -> usize {
  |        ^^^

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0308`.
